src/Endpoints/DNS.php

```diff
@@ -125,6 +125,15 @@ public function getRecordDetails(string $zoneID, string $recordID): \stdClass
         return $this->body->result;
     }
 
+    public function getRecordID(string $zoneID, string $type = '', string $name = ''): string
+    {
+        $records = $this->listRecords($zoneID, $type, $name);
+        if (isset($records->result{0}->id)) {
```

Comment:
Consider following code snippets:

Since the package requires `php-7.0` version it can use this code:

```php
return $records->result{0}->id ?? false;
```